Resources/doc/installation.rst

```diff
@@ -17,7 +19,7 @@ Installation
 
 Add the following snippets to "deps" files:
 
-.. code-block::
+.. code-block:: ini
```

Comment:
please drop the deps file instructions instead of fixing them. they make no sense anymore (the only reason they are still there is because this doc was not used anywhere for real)